changelog/update-10172-csv-export-tracks-event

```diff
@@ -0,0 +1,5 @@
+Significance: patch
+Type: dev
+Comment: Use a single tracks event to record CSV exports, and a location prop to distinguish where and for what the export is generated.
```

Comment:
Might as well mention the event name here 😀

Reply:
Done! 6879338